grouper/ctl/sync_db.py

```diff
@@ -17,10 +18,36 @@ def sync_db_command(args):
 
     # Add some basic database structures we know we will need if they don't exist.
     session = make_session()
+    # This group is needed to bootstrap a Grouper installation.
+    admin_group = Group.get(session, name="grouper-administrators")
+    if not admin_group:
+        admin_group = Group(
+                groupname="grouper-administrators",
+                description="Administrators of the Grouper system.",
+                canjoin="nobody",
+        )
+
+        try:
+            admin_group.add(session)
+            session.flush()
+        except IntegrityError:
+            session.rollback()
+            raise Exception('Failed to create group: grouper-administrators')
+    admin_group_permission_names = [perm[1] for perm in admin_group.my_permissions()]
```

Comment:
There's `Model.get_or_create()` which could replace this code but has the unfortunately side effect that if anyone changed the `description` or `canjoin` attributes would fail trying to create the group again.

Do we want to enforce either of these attributes as immutable? Seems silly but just asking the question. :)